.azure-pipelines/cuda-unit-test.yml

```diff
@@ -30,7 +30,7 @@ steps:
   - script: |
       SB_MICRO_PATH=$PWD python3 setup.py test
     displayName: Run unit tests
-    timeoutInMinutes: 10
+    timeoutInMinutes: 15
```

Comment:
Why need to change it?

Reply:
the timeout is not enough because I add some python package installation